test/SlowTests/Client/Counters/CountersCrudMultipuleNodes.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations.Counters;
+using Raven.Client.Http;
+using Raven.Tests.Core.Utils.Entities;
+using Tests.Infrastructure;
+using Xunit;
+
+namespace SlowTests.Client.Counters
+{
+    public class CountersCrudMultipuleNodes : ClusterTestBase
+    {
+        private static readonly Guid TombstoneMarker = new Guid("DEAD0000-5A81-4CB1-9E4D-E60B8EBDCE64");
+        private const string DocId = "users/1-A";
+
+        [Fact]
+        public async Task IncrementCounter()
```

Comment:
This test is racy, we need to _wait_ until the value is 30.
For that matter, we probably want to wait on _all_ the nodes to converge on 30.